lightningd/plugin.c

```diff
@@ -0,0 +1,35 @@
+#include "lightningd/plugin.h"
+
+#include <ccan/list/list.h>
+#include <ccan/tal/str/str.h>
+
+struct plugin {
+	int stdin, stdout;
+	pid_t pid;
+	char *cmd;
+};
+
+struct plugins {
```

Comment:
I think this should simply be a struct list_head....

Reply:
We add some more state later on, which is why we have a struct in this case.